Mlem/App/Views/Shared/Search/SearchView.swift

```diff
@@ -33,9 +33,9 @@ struct SearchView: View {
     @State var selectedTab: Tab = .communities
     @State var resultsScrollToTopTrigger: Bool = false
     
-    @State var communities: [Community2] = []
-    @State var people: [Person2] = []
-    @State var instances: [InstanceSummary] = []
+    @MainActor @State var communities: [Community2] = []
+    @MainActor @State var people: [Person2] = []
+    @MainActor @State var instances: [InstanceSummary] = []
```

Comment:
I'm now getting 

`Main actor-isolated property 'communities' can not be referenced from a non-isolated context`

on anything that touches these (L119, 123, 127, 140-142, 145, 147, 149) when I try to build on Xcode 15

Reply:
Hmm, maybe it's an Xcode 16 thing then. I'll add the `set...` methods instead 👍

Reply:
Done